Project.toml

```
@@ -9,6 +9,7 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 
 [compat]
 julia = "1"
+PyCall = ">= 1.90"
```

```suggestion
PyCall = "1.90"
```

We aren't sure what will be in PyCall 2.0, so let's just lower bound it.